backend/db/migrate/20221123101219_create_users.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table(:users, id: :uuid) do |t|
+      t.string(:username)
+      t.string(:password_digest)
+
+      t.timestamps
+    end
+
+    add_index(:users, :username, unique: true)
```

Comment:
I just realized. Users to not have an email? Username lang?

Reply:
hmmmmm. right, forgot to ask this. 
Was following the pattern of `bookshelf.lol` regards to registration. 

Planning to migrate din naman to `email` for registration sa next PRs para din may `confirmation` na mangyari.